parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -450,6 +450,11 @@ MASTER_ARTIFACTS_CONFIG_PLACEHOLDER
     sed -i "s|<masqLinkLocalValue>|false|g" "/etc/kubernetes/addons/ip-masq-agent.yaml"
 {{end}}
 
+{{if IsMooncake}}
+    sed -i "s/k8s.gcr.io/gcr.azk8s.cn\/google-containers/g" /etc/kubernetes/addons/*.yaml
```

Comment:
`k8s.gcr.io` would redirect to `gcr.io/google-containers`, so we replace with `gcr.azk8s.cn/google-containers`